deploy/olm-catalog/3scale-operator/3scale-community-operator.v0.4.0.clusterserviceversion.yaml

```diff
@@ -164,6 +163,7 @@ spec:
                 - name: ZYNC_IMAGE
                   value: "quay.io/3scale/3scale27:zync-3scale-2.7.0-GA"
                 image: quay.io/3scale/3scale-operator:v0.4.0
+                imagePullPolicy: Always
```

Comment:
why? 

This tag is not floating tag and default pull policy `IfNotPresent` is good enough

Reply:
This was to match what's in the 2.7-stable-prod branch

Reply:
If the tag is not floating, I would go for default pull policy

Reply:
:+1: done